packages/flask/src/UICountryPicker/CountryPicker.tsx

```diff
@@ -0,0 +1,220 @@
+    }, [countriesList]);
+
+    React.useEffect(() => {
+        const result = search ? fuse.search(search) : countriesList;
+        setFilteredList(result as CountriesArray);
+    }, [search]);
+
+    React.useEffect(() => {
+        fetchJSON()
+            .then((r: CountriesArray) => {
+                if (permitted.length || banned.length) {
+                    filterCountries(r);
+                } else {
+                    setCountriesList(r);
+                }
+            })
+            .catch((e: Error) => {
+                console.error(e);
+            })
+            .finally(() => {
+                setLoading(false);
+            });
+    }, []);
+
+    const renderSearchHeader = () => {
+        return (
+            <View style={styles.headerContainer}>
+                <View style={styles.headerTitleContainer}>
+                    <View style={styles.sideHeaderView}>
+                        <UILinkButton onPress={onClose} title={uiLocalized.Cancel} />
+                    </View>
+                    <UILabel role={TypographyVariants.HeadlineHead} style={styles.headerTitle}>
+                        {uiLocalized.CountryPicker.Title}
+                    </UILabel>
+                    <View style={styles.sideHeaderView} />
+                </View>
+                <UISearchBar returnKeyType="done" value={search} onChangeText={setSearch} />
+            </View>
+        );
+    };
+
+    const renderCountryRow = ({ item }: { item: Country }) => {
+        const onPress = () => onSelectCountry(item);
+        return (
+            <TouchableOpacity onPress={onPress} style={styles.rowContainer}>
+                <View style={styles.rowContainerInner}>
+                    <UILabel>{item.name}</UILabel>
+                    <UILabel style={styles.emojiContainer}>{item.emoji}</UILabel>
+                </View>
+            </TouchableOpacity>
+        );
+    };
+
+    const renderLoading = () => {
+        return <ActivityIndicator />;
+    };
+
+    const renderEmptyList = () => {
+        return (
+            <>
+                <UILabel role={TypographyVariants.TitleMedium} color={ColorVariants.TextSecondary}>
+                    {uiLocalized.CountryPicker.CountryNotFindTitle}
+                </UILabel>
+                <UILabel role={TypographyVariants.Action} color={ColorVariants.TextSecondary}>
+                    {uiLocalized.CountryPicker.CountryNotFindDetails}
+                </UILabel>
+            </>
+        );
+    };
+
+    const ListEmptyComponent = React.useMemo(() => {
+        return (
+            <View style={styles.emptyContainer}>
+                {loading ? renderLoading() : renderEmptyList()}
+            </View>
+        );
+    }, [loading]);
+
+    const keyExtractor = React.useCallback((item: Country) => item.code, []);
+
+    return (
+        <UIBottomSheet onClose={onClose} visible={visible} style={styles.sheet}>
+            {renderSearchHeader()}
+            <FlatList
```

Comment:
Check this out https://github.com/tonlabs/TON-Surf/blob/3cbf5f4ddb62020683526bff9b8c4289d0f4ecae/src/scenes/Chats/components/ListOfThreads.tsx#L308 , this way you can add a space in the bottom to be able to scroll over keyboard :)